packages/lesswrong/lib/collections/curationNotices/permissions.ts

```diff
@@ -0,0 +1,9 @@
+import { sunshineRegimentGroup } from "../../permissions";
+import { adminsGroup } from "../../vulcan-users";
+
+const actions = [
+  'curationNotices.edit.all',
+];
+
+adminsGroup.can(actions);
+sunshineRegimentGroup.can(actions)
```

Comment:
This is a pretty legacy way of doing permissions and I wouldn't use it in new code.  Prefer to instead set custom options for `getDefaultMutations` (you want `newCheck` and `editCheck` to gate it using `userIsAdminOrMod`, and `deleteCheck` to just return `false`), as well as a `checkAccess` function on the entire collection to control who can read curation notices.

Reply:
Done